azure-pipelines.tools.yml

```diff
@@ -1,13 +1,13 @@
 # Install versions of Node and Yarn required by build pipelines.
 steps:
-- task: NodeTool@0
-  inputs:
-    versionSpec: '10.x'
-  displayName: 'Install Node.js'
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
 
-- task: geeklearningio.gl-vsts-tasks-yarn.yarn-installer-task.YarnInstaller@3
-  displayName: 'Use Yarn 1.19.x'
-  inputs:
-    versionSpec: 1.19.x
-    checkLatest: true
-    includePrerelease: false
+  - task: geeklearningio.gl-vsts-tasks-yarn.yarn-installer-task.YarnInstaller@3
+    displayName: 'Use Yarn 1.19.x'
+    inputs:
+      versionSpec: 1.19.x
+      checkLatest: true
+      includePrerelease: false
```

Comment:
What is doing all of this formatting? I was following existing formatting **and** [Azure docs formatting](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml) when I made these changes.

Reply:
The indentation in other files seems excessive. The `-` serves as an indent marker. It'd be kinda like writing code like this:

```tsx
if (test)
  {
    some code;
  }
else
  {
    more code;
  }
```

Reply:
meh - this is prettier yaml stuff - I could add this stuff to the prettierignore?

Reply:
One thing I'm not sure you caught is that I'm moving the steps one level DOWN - so that will make all of it indent one more. So it used to be jobs -> steps, and I made it to be stages -> jobs -> steps to get that visualization showing up on the builds